--- sipp-3.5.1/configure.ac~	2016-03-17 09:05:49.000000000 +0100
+++ sipp-3.5.1/configure.ac	2018-09-19 11:05:52.515888932 +0200
@@ -143,7 +143,7 @@ if test "$openssl" = 'yes'; then
     AC_CHECK_HEADERS([openssl/rand.h],,[AC_MSG_ERROR([<openssl/rand.h> header missing])])
     AC_CHECK_HEADERS([openssl/ssl.h],,[AC_MSG_ERROR([<openssl/ssl.h> header missing])])
     AC_CHECK_HEADERS([openssl/x509v3.h],,[AC_MSG_ERROR([<openssl/x509v3.h> header missing])])
-    AC_CHECK_LIB([ssl], [SSL_library_init],,[AC_MSG_ERROR([ssl library missing])])
+    AC_CHECK_LIB([ssl], [SSL_accept],,[AC_MSG_ERROR([ssl library missing])])
     AC_CHECK_LIB([crypto], [CRYPTO_num_locks],,[AC_MSG_ERROR([crypto library missing])])
 fi
 # For Makefile.am
--- sipp-3.5.1/configure.ac~	2018-09-19 11:07:59.000000000 +0200
+++ sipp-3.5.1/configure.ac	2018-09-19 11:10:01.516629476 +0200
@@ -144,7 +144,7 @@ if test "$openssl" = 'yes'; then
     AC_CHECK_HEADERS([openssl/ssl.h],,[AC_MSG_ERROR([<openssl/ssl.h> header missing])])
     AC_CHECK_HEADERS([openssl/x509v3.h],,[AC_MSG_ERROR([<openssl/x509v3.h> header missing])])
     AC_CHECK_LIB([ssl], [SSL_accept],,[AC_MSG_ERROR([ssl library missing])])
-    AC_CHECK_LIB([crypto], [CRYPTO_num_locks],,[AC_MSG_ERROR([crypto library missing])])
+    AC_CHECK_LIB([crypto], [CRYPTO_free],,[AC_MSG_ERROR([crypto library missing])])
 fi
 # For Makefile.am
 AM_CONDITIONAL(HAVE_OPENSSL, test "$openssl" = "yes")
